components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkConstants.java

```diff
@@ -109,6 +109,7 @@
     public static final String SECRET_KEY_CLAIM_URL = "http://wso2.org/claims/identity/secretkey";
     public static final String IDP_RESOURCE_ID = "IDPResourceID";
     public static final String ENABLE_JIT_PROVISION_ENHANCE_FEATURE = "JITProvisioning.EnableEnhancedFeature";
+    public static final String ERROR_CODE_INVALID_ATTRIBUTE_UPDATE = "SUO-10000";
```

Comment:
Can't we reuse the exiting defined error code constant.

Reply:
Since the constant is in the SCIM component, we can't use it.